test/integration/steps/test_parent_steps.py

```diff
@@ -408,7 +396,11 @@ def the_change_event_is_sent_with_duplicate_sequence(context):
     context["start_time"] = dt.today().timestamp()
     context["correlation_id"] = generate_correlation_id()
     odscode = context["change_event"]["ODSCode"]
-    seqid = get_latest_sequence_id_for_a_given_odscode(odscode)
+    seqid = 0
+    if context["sequenceid"] == 100:
```

Comment:
Can this be a function as I think it gets used in multiple places. Let me know if I'm wrong on this one.

Reply:
Get latest sequence id for an ODS code is a function that's used in a few places. This test uses a context value if there isn't already an ODScode in the DDB, which is really common now because of the way we're randomly picking ODScodes. It didn't used to be as much of a problem, but it was causing intermitted failures.

Reply:
Logic isn't repeated elsewhere